README.md

```diff
@@ -26,3 +26,50 @@ apply plugin: 'org.smartregister.plugin.gradle.android-release-signing-plugin'
 ```
 
 NOTE: This plugin assumes that your Android project has its release keystore uploaded to Travis CI as `android_release.keystore` and the `keystore_password`, `keystore_alias` and `keystore_alias_password` are all set as environment variables in Travis CI.
+
+
+
+radle JarJar Repackage Plugin
+==============================
+
+Plugin for gradle which allows you to repackage jar-libraries with different package names using [JarJar][1] tool.
+
+In Android development there are some situations when library packaged inside your *.apk-file already exists on the device firmware.
+In such case java class loader prefer device's library version instead of your own packaged inside an *.apk.
+If you using newest version of library than that that exists on a device you can stuck in a trouble.
+Your newest library version can have changed signatures which can cause a runtime exception.
+
+For example on some HTC devices there are pre-installed GSON library for sereializing/desereializing to/from JSON.
```

Comment:
Please fix the typo `serializing/deserializing`.

Reply:
done